crypto/.jvm/src/main/scala/org/bitcoins/crypto/BouncyCastleUtil.scala

```diff
@@ -26,16 +26,18 @@ object BouncyCastleUtil {
     decodePubKey(point, publicKey.isCompressed)
   }
 
-  def decodePoint(bytes: ByteVector): ECPoint = {
+  private[crypto] def decodePoint(
+      bytes: ByteVector): org.bouncycastle.math.ec.ECPoint = {
     curve.decodePoint(bytes.toArray)
   }
 
-  def decodePoint(pubKey: ECPublicKey): ECPoint = {
+  private[crypto] def decodePoint(
+      pubKey: ECPublicKey): org.bouncycastle.math.ec.ECPoint = {
     decodePoint(pubKey.bytes)
   }
 
-  def decodePubKey(
-      point: ECPoint,
+  private[crypto] def decodePubKey(
```

Comment:
Is there a reason these are now private?

Reply:
We don't want to expose Bouncy Castle outside.